test/config/config.go

```diff
@@ -39,6 +39,7 @@ type CiliumTestConfigType struct {
 	Kubeconfig          string
 	Registry            string
 	Benchmarks          bool
+	UpdateTestImageTag  string
```

Comment:
Do we need this? We know the target image because it gets set by the image/tag values. I would prefer to reuse that than to introduce a new way to set this

Reply:
Fixed, good catch